core/src/com/unciv/ui/newgamescreen/PlayerPickerTable.kt

```diff
@@ -47,7 +50,12 @@ class PlayerPickerTable(val previousScreen: PreviousScreenInterface, var gamePar
         if (gameParameters.players.count() < ruleset.nations.values.count { it.isMajorCiv() }
                 && !locked) {
             playerListTable.add("+".toLabel(Color.BLACK, 30).apply { this.setAlignment(Align.center) }
-                    .surroundWithCircle(50f).onClick { gameParameters.players.add(Player()); update() }).pad(10f)
+                    .surroundWithCircle(50f).onClick {
+                        var player = Player()
+                        if (noRandom) { player.apply { chosenCiv = getAvailablePlayerCivs().first().name } }
```

Comment:
I'm seeing a LOT of Player().apply { chosenCiv = ... },
I think the correct action would be to add an optional chosenCiv parameter to the Player() constructor

Reply:
Done. Will be fixed in the next commit.